src/cpp/src/llm_pipeline.cpp

```diff
@@ -656,6 +690,35 @@ ov::genai::LLMPipeline::LLMPipeline(
     m_pimpl->m_load_time_ms = std::chrono::duration_cast<std::chrono::milliseconds>(stop_time - start_time).count();
 }
 
+ov::genai::LLMPipeline::LLMPipeline(
+    const std::string& model_str,
+    const ov::Tensor& weights_tensor,
+    const ov::genai::Tokenizer& tokenizer,
+    const std::string& device,
+    const ov::AnyMap& config,
+    const ov::genai::GenerationConfig& generation_config
+){
+    auto start_time = std::chrono::steady_clock::now();
+    if (config.find(ov::genai::scheduler_config.name()) != config.end()) {
+        auto [plugin_config, scheduler_config] = utils::split_scheduler_config(config);
+        m_pimpl = std::make_unique<ContinuousBatchingAdapter>(model_str, weights_tensor,
+                                                              tokenizer, scheduler_config, device, plugin_config, generation_config);
+    } else if ("NPU" == device) {
+        ModelConfigDesc model_descr;
+        m_pimpl = std::make_unique<StaticLLMPipeline>(model_str, weights_tensor, model_descr,
+                                                      tokenizer, device, config, generation_config);
+    } else {
+        auto [core_properties, compile_properties] = ov::genai::utils::split_core_complile_config(config);
+        ov::Core core;
+        core.set_property(core_properties);
+        
+        auto model = core.read_model(model_str, weights_tensor);
```

Comment:
`read_model_with_config` ?

Reply:
`read_model_with_config` works only with files.
I could've overload it for str/tensor reading, but indeed for them no core params is necessary since MMAP is not used for them